### PR TITLE
Added correct addOffer using Redux

### PR DESCRIPTION
**Pull request by Bartosz97**

### CODE REVIEW

**Comment by EnjoyTheNoise** [comment]
First of all: you didn't merge current dev into your branch
Second: did you test it?
Another thing is, you're redirecting by pushing using withRouter and history prop, while in dev we already have method of redirecting using connected-react-router. Can you explain why?

**Comment by Bartosz97** [comment]
1. Now, I pull the newest change from dev.
2. Yes, add offer work correctly

**Review by EnjoyTheNoise — CHANGES_REQUESTED** [review]
Again problems with package-lock, resolve it or don't create PR

File: src/actions/addOfferAction.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+const postNewOfferSuccess = response => ({
+  type: POST_NEW_OFFER_SUCCESS,
+  payload: response.data
+});
+
+const postNewOfferFailure = error => ({
+  type: POST_NEW_OFFER_FAILURE,
+  error
+});
+export const getAllCategories = props => dispatch => {
```
Comment (EnjoyTheNoise) [inline src/actions/addOfferAction.js:46]: what is the point of passing props in every single action?

File: src/actions/httpErrorAction.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
+export const HTTP_401_ERROR = "HTTP_401_ERROR";
```
Comment (EnjoyTheNoise) [inline src/actions/httpErrorAction.js:1]: I know exactly from where did you get this code, but friendly reminder: we're using connected-react-router [https://github.com/supasate/connected-react-router/blob/master/FAQ.md#how-to-navigate-with-redux-action](url)

**Pull request merged**